workspaces/simon-game/src/app/components/App.tsx

```diff
@@ -1,14 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 
 import { Box } from "./Box.tsx";
 import { playNote } from "../util/playNote.ts";
 import { config } from "../constants.ts";
 
 export function App() {
+  const [playerMoves, setPlayerMoves] = useState<number[]>([]);
   return (
     <div style={{ display: "flex", gap: 10 }}>
-      {config.boxes.map((box) => (
-        <Box color={box.color} onClick={() => playNote(box.frequency)} />
+      {config.boxes.map((box, index) => (
+        <Box
+          color={box.color}
+          onClick={() => {
+            playNote(box.frequency);
+            setPlayerMoves([...playerMoves, index]);
```

Comment:
Prefer setting state using previous state when possible. I realize this will make `playerMoves` an unused variable but we'll fix that in a moment.